packages/shared/src/dto/subscriber/subscriber.dto.ts

```diff
@@ -55,4 +55,11 @@ export interface ISubscriberResponseDto {
   createdAt: string;
   updatedAt: string;
   __v?: number;
+  timezone?: string;
 }
+
+export type SubscribersListResponseDto = {
+  data: Array<ISubscriberResponseDto>;
+  next: string | null;
+  previous: string | null;
+};
```

Comment:
had to manually define it here instead of using the one from the SDK because the SDK is mapping the `_id` type to `id` while the server sends `_id` in the response
this makes the FE unable to use the correct property (_id) received in the response.